INSTALL.rst

```diff
@@ -279,3 +280,18 @@ There are no specific commands for the project. See
 ``python src/manage.py --help``.
 
 .. _Django framework commands: https://docs.djangoproject.com/en/dev/ref/django-admin/#available-commands
+
+
+Typical installation errors
+===========================
+l. **Could not find the GDAL library**
+GDAL library is used by GeoDjango. It can be installed separately.
+
+For Ubuntu OS:
+Use external repository UbuntuGIS and install GDAL from it:
```

Comment:
I think this is not needed on Ubuntu, but you rather need the following:

1. `sudo apt-get install libgdal-dev` - to install libgdal (https://packages.ubuntu.com/search?keywords=libgdal-dev) with development headers
2. I don't think you need https://pypi.org/project/GDAL/ in your environment because Django directly interacts with it. I definitely don't remember installing it, and if it's needed, it would've been added to `requirements/base.in` (see also https://docs.djangoproject.com/en/2.1/ref/contrib/gis/install/geolibs/)

Reply:
You are right,  I need just libgdal-dev package, so I removed section with typical errors and added GDAL package as prerequisite